WBCore/GeolocationHelper.swift

```diff
@@ -0,0 +1,188 @@
+//
+//  GeolocationHelper.swift
+//
+
+import WebKit
+import CoreLocation
+
+class GeolocationHelper: NSObject, WKScriptMessageHandler, CLLocationManagerDelegate {
+
+    var locationManager = CLLocationManager();
+    var listenersCount = 0;
+    var webView: WKWebView!;
+
+    override init() {
+        super.init();
+        locationManager.delegate = self;
+    }
+
+    func setUserContentController(webViewConfiguration: WKWebViewConfiguration) {
+        webViewConfiguration.userContentController.add(self, name: "listenerAdded");
+        webViewConfiguration.userContentController.add(self, name: "listenerRemoved");
+    }
+
+    func setWebView(webView: WKWebView) {
+        self.webView = webView;
+    }
+
+    func locationServicesIsEnabled() -> Bool {
+        return (CLLocationManager.locationServicesEnabled()) ? true : false;
+    }
+
+    func authorizationStatusNeedRequest(status: CLAuthorizationStatus) -> Bool {
+        return (status == .notDetermined) ? true : false;
```

Comment:
for all of these ternaries can we just do

```
return status == .notDetermined
```
instead of
```
return (status == .notDetermined) ? true : false;
```
etc., which has the same effect anyway and is neater.